### PR TITLE
Server side filters

### PR DESCRIPTION
**Pull request by bengerman13**
## Changes Proposed

- Add server-side filtering to requests

## Security Considerations

Adds some protection against authenticated but unauthorized changes

### CODE REVIEW

**Review by soutenniza — DISMISSED** [review]
Looks reasonable to me +1

File: src/kibana-cf_authentication/server/filterPath.test.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+const each = require("jest-each").default
```
Comment (apburnes) [inline src/kibana-cf_authentication/server/filterPath.test.js:1]: We will need to install the `jest-each` module in the testing task in the `cg-deploy-logsearch` pipeline to pass to build the release.

**Review by apburnes — COMMENTED** [review]
Just a question about adding the `jest-each` dep for the testing task

File: src/kibana-cf_authentication/server/filterPath.test.js (hunk 1 above)
Comment (bengerman13) [reply src/kibana-cf_authentication/server/filterPath.test.js:1]: I was wondering about that. Do you know an easy way to do that?

Comment (apburnes) [reply src/kibana-cf_authentication/server/filterPath.test.js:1]: We can install in the task but will it add the dependencies to the input repository resource and pass them along to the build release step? Or does the build release get a fresh instances of the input repository resource?

Comment (bengerman13) [reply src/kibana-cf_authentication/server/filterPath.test.js:1]: yeah, we need to move the test task to a new job so the dependencies will be cleared out before the build

Comment (apburnes) [reply src/kibana-cf_authentication/server/filterPath.test.js:1]: Job for installing and testing dev deps merged and pipeline updated https://github.com/cloud-gov/cg-deploy-logsearch/pull/252

**Review by apburnes — APPROVED** [review]

File: src/kibana-cf_authentication/server/filterPath.test.js (hunk 1 above)
Comment (bengerman13) [reply src/kibana-cf_authentication/server/filterPath.test.js:1]: Thanks @apburnes !

**Pull request merged**